classes/Utils.php

```diff
@@ -10006,4 +10003,29 @@ public function get_remote_plugin_info( $plugin_slug = 'tutor' ) {
 
 		return (object) json_decode( $response['body'], true );
 	}
+
+	/**
+	 * Retrieve raw prices for a given course.
+	 *
+	 * This method fetches the regular price and sale price for a specified course 
+	 * from the post meta. It returns the prices as a stdClass object with properties 
+	 * for price and sale_price.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param int $course_id The ID of the course to retrieve prices for.
+	 * @return object An object containing the price and sale price of the course.
+	 *                   - float $price: The regular price of the course.
+	 *                   - float $sale_price: The sale price of the course.
+	 */
+	public function get_course_raw_prices( $course_id ) {
```

Comment:
- Do not add new method. Use this existing one `get_raw_course_price`
- add course price when `monetize_by` tutor.
- Use course price meta constant
```php
Course::COURSE_PRICE_META
Course::COURSE_SALE_PRICE_META
```

Reply:
@harunollyo 
I cannot find Course::COURSE_PRICE_META in my code because the branch is created form `ecommerce-backend`

Reply:
Do pull from `v3`